.github/workflows/lib-injection.yml

```diff
@@ -58,9 +58,10 @@ jobs:
       uses: actions/checkout@v4
       with:
           repository: 'DataDog/system-tests'
+          persist-credentials: false
```

Comment:
:arrow_down: I think we're missing the same change in test-negative, below.

Reply:
Good catch! Updated